miner/README.org

```diff
@@ -25,24 +25,24 @@
 
 * What is Mining?
 
-A blockchain is a series of /blocks/. Blocks contain /transactions/ which
+A blockchain is a series of /blocks/. Blocks contain /transactions/ that
 represent coin transfers or other Smart Contract interactions. Mining is the
 process of mathematically "solving" a block. Unless solved, a block cannot be
 included in the chain. Without mining, a blockchain cannot progress and
 transactions will never finalize.
 
-Economically, mining is the principal way to obtain currency on a Chainweb
-network. Since Chainweb uses Proof-of-Work, computational effort is rewarded
+Economically, mining is the principal way to obtain currency on a Kadena
+network. Since Kadena uses Proof-of-Work, computational effort is rewarded
 with coins associated with each block you solve. The more computational power
 you have, the more likely you are to be successful at mining.
 
-Unlike blockchains that have a single chain, Chainweb has many. Everyone who
+Unlike blockchains that have a single chain, the Kadena public blockchain has many. Everyone who
```

Comment:
Seven syllables in "Kadena public blockchain" is a lot more than the two from "Chainweb".

Reply:
This is okay, as we are wanting to make the transition from 'Chainweb' to 'Kadena public blockchain.'